### PR TITLE
Cards with same name defects

### PR DESCRIPTION
**Pull request by slyjeff**
Had multiple issues where the app was getting confused with Cards that had the same name- the Return to Boundary Beyond showed me this. I fixed this (along with one issue along the way where it wasn't resolving local images for encounter cards).

### CODE REVIEW

File: ArkhamOverlay/CardButtons/IHasImageButton.cs [hunk 1]
```diff
@@ -3,6 +3,8 @@
 namespace ArkhamOverlay.CardButtons {
     public interface IHasImageButton {
         string Name { get; }
+        //string Code { get; }
```
Comment (davilimap) [inline ArkhamOverlay/CardButtons/IHasImageButton.cs]: Nit: remove this if it's unused.

File: ArkhamOverlay/Data/CardInfo.cs [hunk 2]
```diff
@@ -18,6 +18,7 @@ public class CardInfo : IHasImageButton {
 
         public CardInfo(ArkhamDbCard arkhamDbCard, int count, bool isPlayerCard, bool cardBack = false, bool isBonded = false) {
             Code = arkhamDbCard.Code;
```
Comment (davilimap) [inline ArkhamOverlay/Data/CardInfo.cs:20]: ImageId and Code are getting a little mixed here. Is there a reason to keep both?

**Review by davilimap — APPROVED** [review]

File: ArkhamOverlay/Data/CardInfo.cs (hunk 2 above)
Comment (slyjeff) [reply ArkhamOverlay/Data/CardInfo.cs:20]: I'm not sure- Code is the actualy code in ArkhamDB- I originally was modifying it like I was using ImageID, but that felt dirty, so I created ImageID. We aren't using Code for anything now, so we could remove it, but it does mean something, so not sure if we should remove it or not.

Comment (davilimap) [reply ArkhamOverlay/Data/CardInfo.cs:20]: Is there ever a case where they would be different? If not, then I'd suggest, consolidating under one name.

Comment (slyjeff) [reply ArkhamOverlay/Data/CardInfo.cs:20]: Yes, they are different when the image is the back of the card- then I append "Back" to the code to generate the image ID.

**Pull request merged**